login-fire-form.html

```diff
@@ -324,6 +324,55 @@
       'signedin': '_handleSignIn'
     },
 
+    observers: [
+      '_onAuthReady(_statusKnown)',
+    ],
+
+    _onAuthReady: function(ready) {
+
+      if (ready && navigator.credentials && navigator.credentials.preventSilentAccess && this.autoLogin) {
+        var that = this;
+
+        // The new Credential Management API is available
+        if(that.debug) {
+          console.log('Credential Management API available');
+        }
+
+        // User already signedIn, no need to retrieve credentials
+        if (that.signedIn) {
+          return;
+        }
+
+        navigator.credentials.get({
+          password: true,
+          mediation: 'silent'
+        }).then(function (cred) {
+
+          if(that.debug) {
+            console.log('Cred found');
+            console.log(cred);
+          }
+
+          // Ensure they're not empty
+          if (!cred) {
+            return Promise.resolve();
+          }
+
+          if (that.signedIn) {
```

Comment:
What about join this condition and the other above?

Reply:
Done